app/models/listing_category.rb

```diff
@@ -3,7 +3,9 @@ class ListingCategory < ApplicationRecord
   # We standardized on the latter, but keeping the table name was easier.
   self.table_name = "classified_listing_categories"
 
-  has_many :listings, inverse_of: :listing_category
+  has_many :listings, inverse_of: :listing_category,
```

Comment:
Thanks @rhymes! That we didn't notice this until now seems to indicate that we never use this direction of the association.